packages/quick_actions/quick_actions_android/example/android/app/src/androidTest/java/io/flutter/plugins/quickactionsexample/QuickActionsTest.java

```diff
@@ -99,7 +99,7 @@ public void appShortcutLaunchActivityAfterStarting() {
             .filter(s -> s.getId().equals(firstShortcut.getId()))
             .findFirst()
             .get();
-    Intent dynamicShortcutIntent = dynamicShortcut.getIntent();
+    Intent dynamicShortcutIntent = dynamicShortcut.getIntent().addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
This line is too long and needs to be formatted.

Reply:
Removed this fix attempt and excluded `quick_actions` for now since the test is still flaky.